assets/js/components/BatchEdit/About/About.jsx

```diff
@@ -100,8 +101,12 @@ const BatchEditAbout = () => {
     setBatchDeletes(deleteReadyItems);
     setBatchReplaces({
       descriptiveMetadata: { ...replaceItems, ...multiValues.replace },
-      collectionId: currentFormValues["collectionId"],
     });
+
+    Object.keys(currentFormValues["collection"]).length > 0
```

Comment:
No big deal, but another common way to make this check could be:  `currentFormValues.hasOwnKey("collection")`

Reply:
@adamjarling  I used only to eliminate the worst of cases where collection is empty string like:

<img width="479" alt="Screen Shot 2020-10-19 at 10 52 00 AM" src="https://user-images.githubusercontent.com/72217877/96475007-1e6e3500-11f9-11eb-9762-db5f7979e88e.png">